gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjurePlugin.java

```diff
@@ -395,8 +395,10 @@ private static void setupConjureTypescriptProject(
                             task.dependsOn(compileConjureTypeScript);
                             task.mustRunAfter(compileTypeScript);
                             task.getPackageName()
-                                    .set(compileConjureTypeScript.flatMap(
-                                            CompileConjureTypeScriptTask::getPackageName));
+                                    .set(project.provider(options::get)
+                                            .map(opts -> (String) opts.get("packageName"))
```

Comment:
https://output.circle-artifacts.com/output/job/2a7d3130-d6b9-4e8b-9403-1a6ba3e540c6/artifacts/0/~/artifacts/junit/gradle-conjure/test/classes/com.palantir.gradle.conjure.ConjurePublishTypeScriptTest.html#publishes%20generated%20code

```
Caused by: java.lang.RuntimeException: Unknown property: packageName
	at com.palantir.gradle.conjure.api.GeneratorOptions.get(GeneratorOptions.java:61)
	at com.palantir.gradle.conjure.ConjurePlugin.lambda$setupConjureTypescriptProject$17(ConjurePlugin.java:399)
	at org.gradle.api.internal.provider.TransformBackedProvider.mapValue(TransformBackedProvider.java:73)
	at org.gradle.api.internal.provider.TransformBackedProvider.calculateExecutionTimeValue(TransformBackedProvider.java:57)
	at org.gradle.api.internal.provider.OrElseValueProducer.isMissing(OrElseValueProducer.java:65)
	at org.gradle.api.internal.provider.OrElseValueProducer.visitProducerTasks(OrElseValueProducer.java:53)
	at org.gradle.api.internal.provider.AbstractMinimalProvider.visitDependencies(AbstractMinimalProvider.java:133)
	at org.gradle.api.internal.tasks.CachingTaskDependencyResolveContext$TaskGraphImpl.getNodeValues(CachingTaskDependencyResolveContext.java:114)
	at org.gradle.internal.graph.CachingDirectedGraphWalker$GraphWithEmptyEdges.getNodeValues(CachingDirectedGraphWalker.java:213)
	at org.gradle.internal.graph.CachingDirectedGraphWalker.doSearch(CachingDirectedGraphWalker.java:121)
	at org.gradle.internal.graph.CachingDirectedGraphWalker.findValues(CachingDirectedGraphWalker.java:73)
	at org.gradle.api.internal.tasks.CachingTaskDependencyResolveContext.getDependencies(CachingTaskDependencyResolveContext.java:69)
```

Reply:
yeah, I discovered this only once the test run and pushed a fix in subsequent commit